compiler-rt/lib/asan/asan_rtl.cpp

```diff
@@ -71,16 +71,54 @@ static void CheckUnwind() {
 }
 
 // -------------------------- Globals --------------------- {{{1
-static int asan_inited = 0;
-static int asan_init_is_running = 0;
+#if SANITIZER_WINDOWS
```

Comment:
do we need to #if ?
maybe atomic everywhere?

Reply:
I wasn't sure how the change would be viewed, so I added this for Windows only. If it seems benign, I'll remove the `#if SANITIZER_WINDOWS`